Resources/Prototypes/_NF/Catalog/Cargo/cargo_medical.yml

```diff
@@ -7,3 +7,13 @@
   cost: 3000
   category: cargoproduct-category-name-medical
   group: market
+
+- type: cargoProduct
+  id: ChemistryDL
+  icon:
+    sprite: Structures/Storage/Crates/chemcrate_secure.rsi
+    state: icon
+  product: CrateChemistryDL
+  cost: 750
```

Comment:
Might honestly make this more expensive - if a base jug had a value of 750 (per welding fuel), and if you only wanted maybe half of the things in here, maybe 2500 or so?  Somewhere between 3 and 4?